Monika-After-Story/MonikaModDev
# Added Nevermind option to long absence farewell

Adds a `Nevermind` option to the long absence farewell, since you couldn't back out before. Also updated exps on the greeting/changed some dialogue here and there on them.

<details><summary><b>Testing</b></summary>
<p>

1. Start long absence farewell
2. Select `Nevermind`
3. You should now be back in the main loop
4. Verify that clicking `Goodbye` doesn't immediately take you to `jump bye_long_absence_2`
</p>
</details>
